src/base/Stopwatch.h

```diff
@@ -10,7 +10,7 @@
 #define SQUID_SRC_BASE_STOPWATCH_H
 
 #include <chrono>
-#include <cstdint>
+#include <cinttypes>
```

Comment:
> Why? because these files are globally included

If these files are globally included (and we are relying on them to be globally included when making decisions about our code), should not this include be _removed_ instead?

```suggestion
```